src/snowcli/plugins/sql/manager.py

```diff
@@ -11,7 +11,11 @@
 
 class SqlManager(SqlExecutionMixin):
     def execute(
-        self, query: Optional[str], file: Optional[Path], std_in: bool
+        self,
+        query: Optional[str],
+        file: Optional[Path],
+        std_in: bool,
+        remove_comments: bool = False,
```

Comment:
`Comments, now are not shown by default` 

Remove comments shouldn't be default True?

Reply:
Done